docs/virtual_machines/tekton_tasks.md

```diff
@@ -130,3 +103,37 @@ for Windows versions, which uses BIOS. More informations about pipeline can be f
 - [Windows efi installer](https://github.com/kubevirt/ssp-operator/blob/main/data/tekton-pipelines/windows-efi-installer-pipeline.yaml) - Pipeline will prepare a template and Windows 11/2k22 datavolume vith virtio drivers installed. User has to provide a link to working Windows 11/2k22 iso file. Pipeline is suitable for Windows versions, which requires EFI (e.g. Windows 11/2k22). More informations about pipeline can be found [here](https://github.com/kubevirt/kubevirt-tekton-tasks/tree/main/examples/pipelines/windows-efi-installer)
 
 - [Windows customize](https://github.com/kubevirt/ssp-operator/blob/main/data/tekton-pipelines/windows-customize-pipeline.yaml) - Pipeline will install SQL server or VS Code in Windows VM. More informations about pipeline can be found [here](https://github.com/kubevirt/kubevirt-tekton-tasks/tree/main/examples/pipelines/windows-customize)
+
+### Using tasks and example pipelines in different namespace
+You can set in which namespace the example pipelines and tasks will be deployed by setting `spec.tektonPipelinesNamespace` or `spec.tektonTasksNamespace`in the HCO CR:
+
+```console
+apiVersion: hco.kubevirt.io/v1beta1
+kind: HyperConverged
+metadata:
+  name: kubevirt-hyperconverged
+  namespace: kubevirt-hyperconverged
+spec:
+  tektonPipelinesNamespace: userNamespace
+  tektonTasksNamespace: userNamespace
+```
+
+or in SSP CR by setting `spec.tektonPipelines.namespace` or `spec.tektonTasks.namespace`:
+```console
+apiVersion: ssp.kubevirt.io/v1beta2
+kind: SSP
+metadata:
+  name: ssp
+  namespace: kubevirt
+spec:
+  tektonPipelines:
+    namespace: kubevirt
+  tektonTasks:
+    namespace: kubevirt
+```
+
+**Note:** The namespace has to exists before doing this change.
+
+If you define different namespace for pipelines and different namespace for tasks, you will have to create [cluster resolver](https://tekton.dev/docs/pipelines/cluster-resolver/) object.
+In case you change the `tektonPipelinesNamespace` attribute, the pipelines will be deployed in that namespace. By default, example pipelines 
+create the result datavolume in `kubevirt-os-images`. In case you would like to create result datavolume in different namespace (by specifying `baseDvNamespace` attribute in pipeline), there will be required additional RBAC permissions (list of all required RBAC permissions can be found [here](https://github.com/kubevirt/ssp-operator/blob/master/data/tekton-pipelines/pipelines-rbac.yaml)).
```

Comment:
```suggestion
!!! note
    - The namespace has to exists before doing this change. <br />
    - If you define different namespace for pipelines and different namespace for tasks, you will have to create [cluster resolver](https://tekton.dev/docs/pipelines/cluster-resolver/) object. <br />
    - In case you change the `tektonPipelinesNamespace` attribute, the pipelines will be deployed in that namespace. <br />
    - By default, example pipelines create the result datavolume in `kubevirt-os-images`. <br />
    - In case you would like to create result datavolume in different namespace (by specifying `baseDvNamespace` attribute in pipeline), there will be required additional RBAC permissions (list of all required RBAC permissions can be found [here](https://github.com/kubevirt/ssp-operator/blob/master/data/tekton-pipelines/pipelines-rbac.yaml)).
```
This was rendering as a single para and it seems as though these are all distinct things users might want to know so I thought it would be useful to give them bullet points. Ignore if they're related.
Also suggesting we use the admonition markup for the admonition for all of it (which is new since the change to material theme)